scripts/prettier.ts

```diff
@@ -25,7 +26,9 @@ switch (process.env.ACTION) {
     }
     break
   default:
-    console.log("usage: ACTION=write|check prettier")
+    console.log(
+      "usage: prettier write|check\nor (*nix):\n       ACTION=write|check prettier\nor (Windows):\n       set ACTION=write|check&&prettier",
```

Comment:
I think we should stick with one way to pass arguments. This script is not likely to ever require a lot of them so a regular positional argument works best on all systems, let's use that.